hydra-node/test/Hydra/HeadLogicSpec.hs

```diff
@@ -364,6 +364,7 @@ getConfirmedSnapshot = \case
 assertNewState :: IsTx tx => Outcome tx -> IO (HeadState tx)
 assertNewState = \case
   NewState st _ -> pure st
+  OnlyEffects effects -> failure $ "Unexpected 'OnlyEffects' outcome: " <> show effects
```

Comment:
This might break some tests as we also use `assertNewState` when we check that the state did not change and is still `st`.